misc/python/materialize/mzcompose/__init__.py

```diff
@@ -94,6 +94,7 @@ def get_default_system_parameters(
         "enable_alter_swap": "true",
         "enable_columnation_lgalloc": "true",
         "enable_compute_chunked_stack": "true",
+        "enable_compute_correction_v2": "true",
```

Comment:
I plan to remove this before merging. Currently the new correction implementation regresses some of the feature benchmarks and while some of that might be expected, I should probably do some profiling before deciding to accept the regressions.